src/bin/cernan.rs

```diff
@@ -243,14 +243,13 @@ fn main() {
             config_topology.insert(config_path.clone(), config.forwards.clone());
         }
     }
-    {
-        let internal_config = &args.internal;
-        config_topology
+    let internal_config = &args.internal;
+    config_topology
             .insert(cfg_conf!(internal_config), internal_config.forwards.clone());
-        if let Some(ref configs) = args.statsds {
-            for (config_path, config) in configs {
-                config_topology.insert(config_path.clone(), config.forwards.clone());
-            }
+
+    if let Some(ref configs) = args.statsds {
+        for (config_path, config) in configs {
+            config_topology.insert(config_path.clone(), config.forwards.clone());
```

Comment:
Most other sources use `config_topology.insert(cfg_conf!(config), config.forwards.clone());` does it make sense to add `cfg_conf!` to any `config_topology.insert`? Just for consistency (also at sinks config above).

Reply:
Please disregard the comment. I was not reading properly.